target/linux/octeon/dts/cn7130_ubnt_edgerouter-4.dts

```diff
@@ -0,0 +1,213 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "cn7130.dtsi"
```

Comment:
I don't see a point in creating this additional cn7130.dtsi. The compatible will be overwritten anyway, so it's actually a no-op.

Reply:
There is work on the way to support more than just the ER-4, namely the ER-12. That one still needs work because of the integrated switch chip. This will also make it more convenient to get the ER-6p running, once we have access to that board.

Reply:
i dropped `cn7130.dtsi` for now.
`cn7130_ubnt_edgerouter-4.dts` now include universal `cn71xx.dtsi`
it is not needed now. we can always add it late if needed with extra customization (when they will actually be needed).

Reply:
> it is not needed now. we can always add it later if extra customization is needed (when it actually is needed).

Yes, thanks. If there a reason to have them, I'm fine with having multiple nested DTSI files later.